src/app/base/components/bricks-integrator/helper/dummy-nodes.ts

```diff
@@ -215,4 +215,7 @@ def vader_sentiment(record):
         elif vs["compound"] <= -0.05:
             return "negative"
     elif MODE == "scores": 
-        return vs`;
\ No newline at end of file
+        return vs`;
+
+
+export const REMOVE_GROUP_NODES = ['gdpr_compliant', 'not_gdpr_compliant'];
```

Comment:
Will this be removed in the future when the whole GDPR topic is removed from bricks? If yes, maybe add a # TODO comment. If we forget about this we will have these terms, which we want to remove, in the source code for a long time